query/graphql/src/main/java/org/codice/ddf/admin/graphql/transform/FunctionDataFetcherException.java

```diff
@@ -14,40 +14,23 @@
 package org.codice.ddf.admin.graphql.transform;
 
 import java.util.List;
-import java.util.Map;
 
-import org.boon.Boon;
 import org.codice.ddf.admin.api.report.ErrorMessage;
 
-import com.google.common.collect.ImmutableMap;
-
 public class FunctionDataFetcherException extends RuntimeException {
 
-    private String functionName;
-    private Map<String, Object> args;
-
     private List<ErrorMessage> customMessage;
```

Comment:
✏️ Nit: this should be named `customMessages` and so should the ctor parameter on line 24.